api/handlers/types_test.go

```diff
@@ -32,3 +32,28 @@ func Test_CreateRetSimpleBlockInfo(t *testing.T) {
 	assert.Equal(t, int64(got.Height), header.Height)
 	assert.Equal(t, got.Reward, header.Reward)
 }
+
+func Test_CreateRetLastblockInfo(t *testing.T) {
+	var lastblockHeight, lastblockTime int64
+	lastblockHeight = 10399
+	lastblockTime = 12
+	//header := newTestDBBlock(t)
```

Comment:
remove this commented line if it's unused any more.

Reply:
ok